src/utils/FilterUtils.js

```diff
@@ -0,0 +1,22 @@
+const FILTER = {
+  ALL: 'all',
+  ACTIVE: 'active',
+  COMPLETED: 'completed',
+};
+
+const filters = [
+  {
+    name: FILTER.ALL, href: '#/', content: 'All',
+  },
+  {
+    name: FILTER.ACTIVE, href: '#/active', content: 'Active',
+  },
+  {
+    name: FILTER.COMPLETED, href: '#/completed', content: 'Completed',
+  },
+];
+
+export const FilterUtils = {
```

Comment:
fixed.

how many people - so many opinions. Earlier I had a mentor who advised to collect separate functionality into such objects and call them with a capital letter because it's not just plain object but an object which creates a namespace.